edge-functions/ab-testing-simple/package.json

```diff
@@ -10,16 +10,16 @@
   "dependencies": {
     "@vercel/examples-ui": "^0.2.5",
     "js-cookie": "^3.0.1",
-    "next": "^12.0.2",
+    "next": "^12.0.8",
     "react": "latest",
     "react-dom": "latest"
   },
   "devDependencies": {
-    "@types/js-cookie": "^3.0.0",
-    "@types/react": "17.0.8",
-    "autoprefixer": "^10.3.7",
-    "postcss": "^8.3.11",
-    "tailwindcss": "^2.2.17",
-    "typescript": "4.3.2"
+    "@types/js-cookie": "^3.0.1",
+    "@types/react": "17.0.38",
+    "autoprefixer": "^10.4.2",
+    "postcss": "^8.4.5",
+    "tailwindcss": "^3.0.16",
```

Comment:
There was a breaking change for Tailwind, which requires some changes: https://tailwindcss.com/docs/upgrade-guide

Reply:
Done! Thanks for pointing it out!